commands/util.go

```diff
@@ -295,6 +297,16 @@ func loadInfoData(ctx context.Context, d *dinfo) error {
 			d.platforms = append(d.platforms, w.Platforms...)
 		}
 		d.platforms = platformutil.Dedupe(d.platforms)
+		inf, err := c.Info(ctx)
+		if err != nil {
+			if st, ok := grpcerrors.AsGRPCStatus(err); ok && st.Code() == codes.Unimplemented {
+				d.version = "N/A"
```

Comment:
I think we should have a fallback for it. Version can be queried by running `--version` on `buildkitd` or from `docker version` for docker driver. We could add this in the driver interface and let it decide.